docs/dev/Calcite.md

```diff
@@ -0,0 +1,26 @@
+# Drill-specific commits in Apache Calcite
+
+Currently, Drill uses Apache Calcite with additional changes, required for Drill. All the commits were left after 
+update from Calcite `1.4.0` to Calcite `1.15.0` and weren't merged to the Calcite's master yet.
```

Comment:
```suggestion
update from Calcite `1.4.0` to Calcite `1.15.0` and weren't merged to the Calcite's master yet since there is no consensus on them in Calcite community.
```